pom.xml

```diff
@@ -1620,6 +1624,67 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>enforce-jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-versions</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireJavaVersion>
+                                            <version>[11.3,)</version>
+                                            <message>Strongbox only supports JDK 11.3, or higher.</message>
+                                            <level>ERROR</level>
+                                        </requireJavaVersion>
+                                    </rules>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>enforce-jdk8</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-versions</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireJavaVersion>
+                                            <version>[1.8,)</version>
```

Comment:
As far as I know, this would mean `1.8` or **ANY** version higher than 1.8 which would include jdk 9, jdk10, jdk11, etc?

Reply:
The jdk version could be upper bounded by jdk 9

Reply:
@steve-todorov : A few lines [above](https://github.com/strongbox/strongbox-parent/pull/103/files#diff-600376dffeb79835ede4a0b285078036R1648), in the profile's activation section, the activation rule says that this should only get triggered for JDK 1.8, so, in theory, it should be fine.

@AricMonary : Could you please test it with JDK 1.8 and 11 and provider sample output as well? (Also, try `mvn help:active-profiles` and share the output, please). Thanks! :)